bibtexparser/tests/test_customization.py

```diff
@@ -22,6 +22,13 @@ def test_getnames(self):
                  'Jean la Tour',
                  'Jean le Tour',
                  'Mike ben Akar',
+                 'A. {Delgado de Molina}',
+                 r'M. Vign{\'e}',
+                 'Tom {de Geus}',
+                 'Tom de {G\{eus}',
+                 'Foo B{\'a}r',
+                 r'{G{\'{e}}rard} {Ben Arous}',
+                 'Incorrect {{name}',
```

Comment:
Small point: Could you add some test cases containing escaped brackets (e.g. `\{ ... \}`). I see in the code that you implemented it, but it would be great to have it covered in the tests.

Otherwise, this LGTM.

Reply:
Done!